website/docs/features/onnx/about.md

```diff
@@ -13,6 +13,33 @@ description: Learn how to use the ONNX model transformer to run inference for an
 
 SynapseML now includes a Spark transformer to bring an trained ONNX model to Apache Spark, so you can run inference on your data with Spark's large-scale data processing power.
 
+## ONNXHub
+While you can use a local model if you have one, many popular existing models are provided through the ONNXHub. You can simply use
+a model's name (e.g. "MNIST") and download the bytes of the model, as well as some metadata about the model. You can also list
+available models, optionally filtering by name or tags. You can also point to your own custom hub uri if you have one.
+
+```scala
+    // List models
+    val hub = new ONNXHub()
+    val models = hub.listModels(model = Some("mnist"), tags = Some(Seq("vision")))
+
+    // Retrieve and transform with a model
+    val info = hub.getModelInfo("resnet50")
+    val bytes = hub.load(name)
+    val model = new ONNXModel()
+      .setModelPayload(bytes)
+      .setFeedDict(Map("data" -> "features"))
+      .setFetchDict(Map("rawPrediction" -> "resnetv24_dense0_fwd"))
+      .setSoftMaxDict(Map("rawPrediction" -> "probability"))
+      .setArgMaxDict(Map("rawPrediction" -> "prediction"))
+      .setMiniBatchSize(1)
+
+    val (probability, _) = model.transform({YOUR_DATAFRAME})
+      .select("probability", "prediction")
```

Comment:
Maybe replace the your data frame placeholder with a simple example def from public blob

Reply:
This is meant to be example of methods/syntax, not an executable piece of code. We can refactor that as part of any work on converting docs to notebooks.